src/libUtils/Logger.cpp

```diff
@@ -278,6 +285,36 @@ void Logger::LogPayload(LEVELS level, const char* msg,
     }
 }
 
+void Logger::LogEpochInfo(const char* msg, const char* function,
+                          const char* epoch)
+{
+    pid_t tid = getCurrentPid();
+
+    auto clockNow = std::chrono::system_clock::now();
+    std::time_t curTime = std::chrono::system_clock::to_time_t(clockNow);
+    auto gmtTime = gmtime(&curTime);
```

Comment:
Please refer to LogEpoch(), only 1 variable is needed instead of 3

Reply:
It's LOG_EPOCHINFO, different from your LOG_EPOCH. it's a stand alone logging function for the usage of daemon, pls remain the same at this stage.

Reply:
Use 1 local variable instead of 3 variables can reduce stack memory usage, and might avoid potential memory problem if LOG_XXX is called many times.
Just need to simply refine as following:

std::time_t curTime = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
...
PAD(put_time(gmtime(&curTime), "%H:%M:%S")

Reply:
ok